form/root_storage/root_rfield_write_container.cpp

```
@@ -0,0 +1,130 @@
+#include "TFile.h"
+
+#include <exception>
+#include <iostream>
+
+namespace form::detail::experimental {
+  ROOT_RField_Write_ContainerImp::ROOT_RField_Write_ContainerImp(std::string const& name) :
+    Storage_Associative_Write_Container(name)
+  {
+  }
+
+  void ROOT_RField_Write_ContainerImp::setAttribute(std::string const& key,
```

Cppcheck (2.20.0)

[style] The function 'setAttribute' is never used.

(unusedFunction)